modules/rest_api/controllers/services/rest.php

```diff
@@ -1044,16 +1044,40 @@ private function checkAllowedResource($proj_id, $resourceName) {
       ['caption' => 'Comment', 'field' => 'occurrence.occurrence_remarks'],
       ['caption' => 'Sample comment', 'field' => 'event.event_remarks'],
       ['caption' => 'Images', 'field' => '#occurrence_media#'],
-      ['caption' => 'Input on date', 'field' => 'metadata.created_on'], // Format ??
-      ['caption' => 'Last edited on date', 'field' => 'metadata.updated_on'], // Format ??
+      ['caption' => 'Input on date', 'field' => '#datetime:metadata.created_on:d/m/Y H^i#'], // Can't use : in format spec here - use ^ instead which is translated to :
+      ['caption' => 'Last edited on date', 'field' => '#datetime:metadata.updated_on:d/m/Y H^i#'], // Can't use : in format spec here - use ^ instead which is translated to :
+      ['caption' => 'Verification status 1', 'field' => '#backward:identification.verification_status#'],
+      ['caption' => 'Verification status 2', 'field' => '#backward:identification.verification_substatus#'],
+      ['caption' => 'Query', 'field' => '#backward:identification.query#'],
+      ['caption' => 'Verifier', 'field' => 'identification.verifier.name'],
+      ['caption' => 'Verified on', 'field' => '#datetime:identification.verified_on:d/m/Y H^i#'], // Can't use : in format spec here - use ^ instead which is translated to :
+      ['caption' => 'Licence', 'field' => 'metadata.licence_code'],
+      ['caption' => 'Automated checks', 'field' => 'identification.auto_checks.result'], 
+    ],
+    "mapmate" => [
+      ['caption' => 'Taxon', 'field' => 'taxon.accepted_name'],
+      ['caption' => 'Site', 'field' => 'location.verbatim_locality'],
+      ['caption' => 'Gridref', 'field' => 'location.output_sref'],
+      ['caption' => 'VC', 'field' => '#mapmate_vc#'],
+      ['caption' => 'Recorder', 'field' => 'event.recorded_by'],
+      ['caption' => 'Determiner', 'field' => 'identification.identified_by'],
+      ['caption' => 'Date', 'field' => '#mapmate_date#'],
+      ['caption' => 'Quantity', 'field' => '#organism_quantity:mapmate#'],
+      ['caption' => 'Method', 'field' => 'event.sampling_protocol'],
+      ['caption' => 'Sex', 'field' => '#mapmate_sex#'],
+      ['caption' => 'Stage', 'field' => '#mapmate_life_stage#'],
```

Comment:
As per other comment about whether to prefix, or have a format specifier as a suffix. For organism_quantity mapmate is a suffix, but not for date, sex, life_stage, vc. Perhaps keeping this consistent would be better, having for example a life_stage special field with mapmate as a format option. Then it makes it more extensible if other format variations get added. Or where these map directly to a single field, it might be cleaner to just add the format specifier to the field name ('occurrence.life_stage:map_mate') for example. I don't have a strong opinion which approach is better though.

Reply:
I went the the suffix route for consistency with the modified 'backward' handling. I didn't implement the format specifier for single fields as it would have required a bigger change and when I thought it through I could see a large block of code that would need to dealt with various combinations of field and format identifers.

Reply:
Ok, thanks - agree there's no need to implement the bigger change required for single field formatters. Let me know when you've done with this Pull Request (if not already?) so I can look at getting it merged.

Reply:
Thanks John - it's ready now. I've tested on dev warehouse.